pkg/webreg/webreg.go

```diff
@@ -611,37 +611,74 @@ Example:
 
 <h3 id="image">How do I use an image from another repo in my repo’s tests?</h3>
 <p>
-All images built for the release targeted by the job (ex:
-<code>release-4.2</code>) are available under <code>stable:tag-name</code>
-(example: <code>stable:installer</code> or <code>stable:cli</code>). There
-are 2 ways of using these images: creating a registry reference that can be
-used in your <code>ci-operator</code> config or adding a literal test step to
-the test in your <code>ci-operator</code> config. Images built by ci-operator
-for your repo are also available under <code>pipeline:tag-name</code>. Here
-are 2 examples of adding a very simple step that runs an oc command on the
-cluster created by <code>ipi-install</code> using the <code>stable:cli</code>
-image:
+In order to use an image from one repository in the tests of another, it is necessary
+to first publish the image to the integration stream from the producer repository and
```

Comment:
Is is supposed to be integration stream or image stream?

Reply:
I call something like `ocp/4.2` an integration stream, e.g. continuous integration